docs/appendices/release-notes/5.5.0.rst

```diff
@@ -51,6 +51,9 @@ Breaking Changes
 - Validation for ``COPY FROM`` is now enforced. The ``validation`` parameter is
   deprecated and using it results in a deprecation message in the logs.
 
+- `UPDATE`` statement shows number of updated rows in case of partial failure
+  instead of always showing an error.
+
```

Comment:
Not sure but I think this belongs to breaking changes list?

Reply:
BTW, Jenkins failure is unrealted, Java tests are passing, it's docs related
`/var/lib/jenkins/workspace/CrateDB/crate_on_pr/docs/appendices/release-notes/5.5.0.rst:149:Failed to create a cross reference. A title or caption not found: memory.operation_limit`